app/javascript/components/student/tracks-list/TagsFilter.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react'
+import { TagOptionList } from './TagOptionList'
+
+export function TagsFilter({ options, dispatch }) {
+  const [hidden, setHidden] = useState(true)
+  const [value, setValue] = useState([])
```

Comment:
I didn't immediately understand what `value` represented. We do use it in several other places, so I can understand why we're using it. Just wanted to put out an alternative:

```suggestion
  const [selectedTags, setSelectedTags] = useState([])
```

or

```suggestion
  const [filterTags, setFilterTags] = useState([])
```